src/main/java/com/emc/mongoose/Server.java

```diff
@@ -0,0 +1,38 @@
+package com.emc.mongoose;
+
+import io.prometheus.client.exporter.MetricsServlet;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+
+/**
+ @author veronika K. on 05.10.18 */
+public class Server {
+
+	private final org.eclipse.jetty.server.Server server;
+
+	public Server(final int port) {
```

Comment:
How do we reuse this for the control API? Control API should be able to add its own specific context.